package.json

```diff
@@ -14,7 +14,7 @@
   },
   "scripts": {
     "start": "node node_modules/grunt-cli/bin/grunt",
-    "test": "node node_modules/grunt-cli/bin/grunt test",
+    "test": "karma start karma.conf.js",
```

Comment:
@gangachris should I use nabs (https://www.npmjs.com/package/nabs) to nest tests tasks or how shoud this be done?